### DISCUSSION TITLE
Server-Translation has no effect on agents-binarys?

### DISCUSSION THREAD

**Discussion by n1smithy** [Ylianst/MeshCentral#6479] [Q&A]
Hey people,
I've made some improvements within the German translation, but there are three things I've noticed:

1. Even after manually updating a Windows-Agent the request-text … OK, my bad. Forgot to click F3 (Upload to server) in the translation-editor. I should go to bed …
2. In all requesting-windows the text-colour-setup within the config.json only has effect on the requesting-text, but not on the text behind the checkbox below.
3. On Linux-systems the requesting-text isn't translated at all. It's in English (tested both on SuSE with KDE and Arch Linux with Gnome/Wayland)

Do I something wrong?

**Comment by si458** [comment]
I'm not 100% sure for the agent translations, but if I remember, u have to manually edit the agent-translation.json file located under the translations folder, then restart meshcentral and let it rebuild the agents

Also, any changes u make inside config.json/translations in relation to the meshagent are not applied to any existing installs!

So you must uninstall the meshagent, redownload the meshagent, and install the meshagent again

**Reply by n1smithy** [reply]
Hey, Simon,

thank you for a quick reply again!

Ahh, manually this json, alright! Do all the agents rebuild just after restarting the server/docker-container? As I edited I found out, the translations for requesting access on windows-systems will be adapted after the „normal“ temporairy server-translation and manual updating the agent.

However, I'll focus on that in a day or two. Today my batteries are low … 😅

I'm goin' to give an update as soon as I tried your information out.

Have a nice weekend!
